### PR TITLE
front-end for SavedConnections page

### PR DESCRIPTION
**Pull request by larahynkim**

### CODE REVIEW

File: front-end/src/SavedConnections.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './SavedConnections.css';
+
+
+const SavedConnections = () => {
+  const connections = [
+    { id: 1, name: 'John Doe' },
```
Comment (XinranTaoAngela) [inline front-end/src/SavedConnections.js:8]: I think for now we need to mock data that will be pulled from the backend using Mockaroo, Sivan mentioned in the chat on how to use that, you can also look at the master branch and my PR for references

**Review by XinranTaoAngela — CHANGES_REQUESTED** [review]
Hi! over all very nice, you need some minor changes, let us know in the chat if theres anything uncertain!

File: front-end/src/SavedConnections.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    { id: 2, name: 'Jane Doe' },
+    { id: 3, name: 'John Doe' },
+    { id: 4, name: 'Jane Doe' },
+  ];
+
+  return (
+    <div className="saved-connections-container">
+      <h2>Saved Connections</h2>
+      <div className="connections-grid">
```
[the remark this answers is not in the document]
Comment (larahynkim) [reply front-end/src/SavedConnections.js:17]: thanks for the feedback! i saw in app.js of the master that the header and footer components are rendered globally across all pages. bc of this, i don't think we have to add the header and footer to each of the pages ??  but lmk if i'm wrong!

File: front-end/src/SavedConnections.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.sc-header {
```
Comment (SivanC) [inline front-end/src/SavedConnections.css:7]: try to keep all the class names unabbreviated

Comment (larahynkim) [reply front-end/src/SavedConnections.css]: thanks! just made the revisions with all of your suggestions

**Review by Pan-LLi — APPROVED** [review]

**Review by XinranTaoAngela — APPROVED** [review]

**Pull request merged**